tools.go

```diff
@@ -0,0 +1,7 @@
+// +build tools
+
+package tools
```

Comment:
Mind adding `vfsgen` to this file? That should satisfy #3279 and then allow us to delete the special handling of `go mod tidy`:

```diff
diff --git a/tools.go b/tools.go
index 0a53843a..8ac42785 100644
--- a/tools.go
+++ b/tools.go
@@ -3,5 +3,6 @@
 package tools

 import (
+       _ "github.com/shurcooL/vfsgen"
        _ "golang.org/x/tools/cmd/goimports"
 )
```

In BUILD.md:

 ```bash
 go mod tidy
-bin/build-cli-bin # adds back dependencies specific to `go generate` commands
 bin/update-go-deps-shas
 ```